src/pages/components/Demo/components/Translate.tsx

```diff
@@ -38,9 +63,32 @@ const Translate = () => {
               resize="none"
             />
           </VStack>
+
+          <Box
+            position={{ base: 'relative', md: 'absolute' }}
+            left={{ md: '50%' }}
+            top={{ md: '60%' }}
+            transform={{ md: 'translate(-50%, -60%)' }}
+            zIndex={1}
```

Comment:
Once we start using flex boxes, we can remove all these fields and opt to use flex box-specific properties like `justifyContent` or `alignItems`